components/topbar/user/src/index.scss

```diff
@@ -21,6 +21,18 @@
   width: $notifications-size;
 }
 
+.html-has-scroll-disabled {
```

Comment:
I was trying to avoid this solution because it creates global classnames, and that's not a good idea, but regarding iOS Safari behaviour, it doesn't accept to write on read-only DOM elements `body` and `html`.